lib/inbox.rb

```diff
@@ -237,32 +237,45 @@ def latest_cursor
       "message" => Inbox::ExpandedMessage,
     }
 
-    def _build_exclude_types(exclude_types)
-      exclude_string = "&exclude_types="
+    def _build_exclude_types(types)
+      exclude_string = "&exclude_types=" + _build_types_list(types)
+    end
+
+    def _build_include_types(types)
+      include_string = "&include_types=" + _build_types_list(types)
+    end
 
-      exclude_types.each do |value|
+    def _build_types_list(types)
+      type_string = ""
+      types.each do |value|
         count = 0
         if OBJECTS_TABLE.has_value?(value)
           param_name = OBJECTS_TABLE.key(value)
-          exclude_string += "#{param_name},"
+          type_string += "#{param_name},"
         end
       end
 
-      exclude_string = exclude_string[0..-2]
+      type_string = type_string[0..-2]
     end
 
-    def deltas(cursor, exclude_types=[], expanded_view=false)
-      return enum_for(:deltas, cursor, exclude_types, expanded_view) unless block_given?
+    def deltas(cursor, exclude_types=[], include_types=[], expanded_view=false)
```

Comment:
If we're willing to break compatibility with Ruby < 2, we can refactor to have the function footprint be:

`deltas(cursor, exclude_types: [], include_types: [], expanded_view: [])`

Which gives us great backward compatibility in the future, at the cost of breaking it now for a subset of people (anyone calling `deltas` with any parameter other than just `cursor` will have issues) and also losing support for older ruby.